src/modules/core/components/Numeral/Numeral.tsx

```diff
@@ -7,7 +7,7 @@ import { formatTokenValue } from '~utils/numbers';
 import styles from './Numeral.css';
 
 interface Appearance {
-  theme: 'primary' | 'grey' | 'dark' | 'blue';
+  theme: 'primary' | 'grey' | 'dark' | 'blue' | 'darken';
```

Comment:
Why is the name of the theme `darken`? It's nitpicking but it sounds bizarre because I think it's a verb. Maybe we can use `darker` instead?

Reply:
I agree with Olga here, `darken` is a bad name.

Maybe come up with something that is not necessarily related to the shade. Maybe something like: `NightGlow` or something equally ridiculous :)

Reply:
because i didint have any other idea xD